ghcide/src/Development/IDE/Plugin/Completions/Logic.hs

```diff
@@ -587,20 +637,33 @@ getCompletions plId ideOpts CC {allModNamesAsNS, anyQualCompls, unqualCompls, qu
           $ (if T.null enteredQual then id else mapMaybe (T.stripPrefix enteredQual))
             allModNamesAsNS
 
-      filtCompls = Fuzzy.filter chunkSize maxC prefixText ctxCompls (label . snd)
-        where
-
-          mcc = case maybe_parsed of
+      -- ----------------------------------------
+      -- Note: correct the cursorPos by moving
```

Comment:
this comment doesn't seem to be in the right place

Reply:
Well, it never was. It used to be:
```Haskell
{- COMMENT -}
pos = cursorPos prefixInfo
```
It probably hints at something going on in the `maybeContext` right below or `filterCompls` below it.